include/tester/vtcp/tcp/TCPServer.hpp

```diff
@@ -41,25 +39,26 @@ class TCPServer
 protected:    
     boost::asio::io_context _context;
 
-    std::unordered_map<int , std::shared_ptr<TCPSession>> _sessions;
+    std::unordered_map<std::size_t , std::shared_ptr<TCPSession>> _sessions;
```

Comment:
Check if you can add a `const` modifier to std::size_t (as you did in the rest of the methods)

Reply:
I can't. In order to add the `const` prefix I need to init `_sessions` in the ctor

Reply:
The error you're getting is `the default constructor of std::unordered_map<const std::size_t, ...> cannot be referenced, it is a deleted function` - From what I've seen there's really no need to put const on an `std::unordered_map` key, so keep it non-const like you have done. 👍